### PR TITLE
Incomplete coverage of test scenarios

### PR DESCRIPTION
**Pull request by Octopusjust**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
The test scenarios is incomplete
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by neolit123** [comment]
please keep the commits squashed to 1.

/priority backlog
/release-note-none

File: cmd/kubeadm/app/preflight/utils_test.go [hunk 1]
```diff
@@ -34,6 +34,14 @@ func TestGetKubeletVersion(t *testing.T) {
...
 		{"Kubernetes v1.7.0", "1.7.0", nil, true},
 		{"Kubernetes v1.8.0-alpha.2.1231+afabd012389d53a", "1.8.0-alpha.2.1231+afabd012389d53a", nil, true},
+		{"Kubernetes v1.8.0-alpha.2.1231+afabd012389d53a ", "1.8.0-alpha.2.1231+afabd012389d53a", nil, true},
+		{" Kubernetes v1.8.0-alpha.2.1231+afabd012389d53a", "1.8.0-alpha.2.1231+afabd012389d53a", nil, true},
+		{"Kubernetes  v1.8.0-alpha.2.1231+afabd012389d53a", "", nil, false},
+		{"Kubernetes v1.8.0alpha.2.1231+afabd012389d53a", "", nil, false},
+		{"Kubernetes v1.8.0--alpha.2.1231+afabd012389d53a", "1.8.0--alpha.2.1231+afabd012389d53a", nil, true},
+		{"Kubernetes v1.8.0-+alpha.2.1231+afabd012389d53a", "", nil, false},
+		{"Kubernetes v1.8.0-alpha.2.1231+-afabd012389d53a", "1.8.0-alpha.2.1231+-afabd012389d53a", nil, true},
+		{"Kubernetes v1.8.0-alpha.2.1231++afabd012389d53a", "", nil, false},
```
Comment (neolit123) [inline cmd/kubeadm/app/preflight/utils_test.go:44]: some of these are not supported by semver and formatting of version in the kubelet - e.g. we assume that `v1.8.0-alpha.2.1231++afabd012389d53a` or `Kubernetes  v1.8.0...` will never happen.

**Comment by danwinship** [comment]
/hold
/remove-sig network
This PR removes and then recreates `pkg/controller/nodeipam/cidrset/cidr_set.go`.
Please make sure the PR contains only the commits you meant for it to include.

File: cmd/kubeadm/app/preflight/utils_test.go (hunk 1 above)
Comment (Octopusjust) [reply cmd/kubeadm/app/preflight/utils_test.go:44]: OK，I will modify it

**Pull request closed without merging**